config/300-ibmmqsource.yaml

```diff
@@ -0,0 +1,225 @@
+    - eventing
+    - sources
+  scope: Namespaced
+  versions:
+  - name: v1alpha1
+    served: true
+    storage: true
+    subresources:
+      status: {}
+    schema:
+      openAPIV3Schema:
+        type: object
+        description: TriggerMesh event source for IBM MQ.
+        properties:
+          spec:
+            description: Desired state of the event source.
+            type: object
+            properties:
+              connectionName:
+                description: IBM MQ server URI, e.g. ibm-mq.default.svc.cluster.local(1414).
+                type: string
+              channelName:
+                description: The name of IBM MQ Channel to listen events on.
+                type: string
+              queueManager:
+                description: IBM MQ Queue Manager name.
+                type: string
+              queueName:
+                description: IBM MQ event source Queue name.
+                type: string
+              delivery:
+                description: Events delivery parameters.
+                type: object
+                properties:
+                  deadLetterQueueManager:
+                    description: The name of the Dead Letter Queue Manager.
+                    type: string
+                  deadLetterQueue:
+                    description: The name of the Dead Letter Queue.
+                    type: string
+                  backoffDelay:
+                    description: Delay between delivery retries for failed messages.
+                    type: integer
+                  retry:
+                    description: The number of delivery retries before moving the message to DLQ.
+                    type: integer
+              sink:
+                description: The destination of events sourced from IBM MQ.
+                type: object
+                properties:
+                  ref:
+                    description: Reference to an addressable Kubernetes object to be used as the destination of events.
+                    type: object
+                    properties:
+                      apiVersion:
+                        type: string
+                      kind:
+                        type: string
+                      namespace:
+                        type: string
+                      name:
+                        type: string
+                    required:
+                    - apiVersion
+                    - kind
+                    - name
+                  uri:
+                    description: URI to use as the destination of events.
+                    type: string
+                    format: uri
+                oneOf:
+                - required: [ref]
+                - required: [uri]
+              credentials:
+                type: object
+                properties:
+                  username:
+                    type: object
+                    properties:
+                      value:
+                        description: Plain text user ID.
```

Comment:
Where does this user ID come from? is there some documentation we can link here?